### DISCUSSION TITLE
Checkbox on Form

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6821] [General]
Hi all,

I have a small int field on db and I need this to be edited with a checkbox (see attached emage)

I tryed to change file xyzForm.ts in "Modules\ServerTypes\Administration\"  in this way:

```
export class CustomerForm extends PrefixedContext {
    static formKey = 'Administration.Customer';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!CustomerForm.init)  {
            CustomerForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;
            var w2 = DateEditor;
            var w3 = CheckboxEditor;

            initFormType(CustomerForm, [
                'Name', w0,
                'Code', w0,
                'Email', w0,
                'TenantName', w1,
                'Enabled', w3,
                'InsertDate', w2,
                'InsertedBy', w1,
                'UpdateDate', w2,
                'UpdatedBy', w1
            ]);
        }
    }
}
```
but when I build and run changes are overwritten.

Is there a way to do it?

Thank you, 
Elena.
![Schermata 2023-07-04 alle 09 57 38](https://github.com/serenity-is/Serenity/assets/14870650/aaa00844-88bd-4c63-a9b2-bc8bdd89aef5)

**Comment by VictorTomaili** [comment]
ServerTypes generated automatically from Form.cs files. Use your editor like an attribute in Form.cs on Abilitato property.

**Reply by elenaaralla** [reply]
Hi

CheckBox editor is not defined :(

What can I do?

![Schermata 2023-07-04 alle 10 55 03](https://github.com/serenity-is/Serenity/assets/14870650/3c60bbc8-4d25-45e5-b1ee-af5016b01809)

**Reply by VictorTomaili** [reply]
try to use `BooleanEditor`

**Reply by elenaaralla** [reply]
Hi @VictorTomaili 

BooleanEditor is the right editor; 
Thank you,
Elena.
